changelog.md

```diff
@@ -31,6 +31,152 @@
 - Removed deprecated `iup` module from stdlib, it has already moved to
   [nimble](https://github.com/nim-lang/iup).
 
+  The following modules now compile on both JS and NimScript: `parsecsv`,
```

Comment:
changelog needs fixing

Reply:
Yeah I noticed it still conflicting, even if it suppose not to :(

Reply:
argue for re-opening https://github.com/nim-lang/Nim/pull/14025

Reply:
I like the idea as starter for a change, but ultimately a script should do it automatically like my example, 
1 tiny script that does 1 thing, I mean [theres even less useful dead stuff in the repo anyways](https://github.com/nim-lang/Nim/blob/devel/tools/nimrepl.nim).